### ISSUE TITLE
App can not be launched from Android Studio because of missing debug keystore

### ISSUE THREAD

**Issue by eggeral** [austrianredcross/stopp-corona-android#15]
## Description

When trying to launch the app from Android studio the build fails because the file keystore/android_debug.keystore can not be found

## Reproduction steps

1. Open the app in Android Studio
2. Hit the run button
Additional helpful information:

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:validateSigningProdApiDebug'.
> Keystore file '................../stopp-corona-android/keystore/android_debug.keystore' not found for signing config 'debug'.

## What did you expect

The app opens in the simulator

**Comment by eggeral** [comment]
Workaround: 

* Remove the debug entry from signingConfigs in order to use the default keystore which was created by Android Studio.

In `.../stopp-corona-android/app/build.gradle`

locate

```
    signingConfigs {
        release {
            keyAlias getKeyAlias()
            storeFile file("../keystore/release.keystore")
            storePassword getStorePassword()
            keyPassword getKeyPassword()
        }

        debug {
            keyAlias "android_debug_key"
            storeFile file("../keystore/android_debug.keystore")
            storePassword "android_debug"
            keyPassword "android_debug"
        }
    }
```
remove
```
 debug {
            keyAlias "android_debug_key"
            storeFile file("../keystore/android_debug.keystore")
            storePassword "android_debug"
            keyPassword "android_debug"
        }
```

**Comment by ckudera** [comment]
Zum Anlegen des Keystore im Hauptverzeichnis folgende Befehle ausführen:
```
mkdir keystore

cd keystore

keytool -genkey -v -keystore android_debug.keystore -storepass android_debug -alias android_debug_key -keypass android_debug -keyalg RSA -keysize 2048 -validity 10000
```

@mitsuhiko Vl. kann man das ins Readme (`Checklist to make project buildable`) aufnehmen?

**Comment by mitsuhiko** [comment]
@ckudera habs mal notiert. ich will jetzt die README nicht auf dem branch aendern weil ich noch nicht weiss wie master am Montag ausschaut. Sonst muss ich alles cherry-picken.

**Comment by marandaneto** [comment]
ideally, the debug build should not even be signed, as the tooling would sign it anyway using `$HOME/.android/debug.keystore`
https://developer.android.com/studio/publish/app-signing#debug-mode
So this could be even removed IMHO

**Comment by uluckas** [comment]
Fixed in bc35c4384f05d4dfd5562aed98c0b0aaa905f782
